twilio/twiml/__init__.py

```diff
@@ -9,6 +10,21 @@ def lower_camel(string):
     return result[0].lower() + result[1:]
 
 
+def format_language(language):
+    """
+    Attempt to format language parameter as 'ww-WW'.
+
+    :param string language: language parameter
+    """
+    if not language:
+        return language
+
+    if not re.match('^\w\w[_-]\w\w$', language):
```

Comment:
`\w` matches words. It seems like we might want to match letters here? Like with `[a-z]` or `[A-Z]`.